hack/run_with_local_webhook.sh

```diff
@@ -90,3 +100,30 @@ webhooks:
 EOF_CAT
 
 oc apply -n openstack -f ${TMPDIR}/patch_webhook_configurations.yaml
+
+oc apply -n openstack -f ${TMPDIR}/patch_webhook_configurations.yaml
```

Comment:
this line looks like it's now duplicated

Reply:
Thanks, I'll fix that

Reply:
Thanks!